community/kernel/src/test/java/org/neo4j/kernel/configuration/SettingsTest.java

```diff
@@ -46,8 +46,9 @@ public void makePathInCurrentWorkingDirectory()
     @Test
     public void doNotModifyAbsolutePaths()
     {
-        File thePath = Settings.PATH.apply( "/some/absolute/path" );
+        File absolutePath = new File( "some/path" ).getAbsoluteFile();
+        File thePath = Settings.PATH.apply( absolutePath.toString() );
```

Comment:
I don't know if this applies to your case, but have you investigated `File.getCanonicalPath`? Absolute path will insert `../` when moving up the directory hierarchy, while canonical path will resolve and simplify such movements.